test/unit/reductions/moments/data_generator.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import numpy as np
+import pandas as pd
+
+
+def simple_binary_threshold_data(number_a0, number_a1,
```

Comment:
I didn't find this self-explanatory in case that was intended. 
Still not quite sure what "a" stands for. I'll have to look at how it's used.
Then further below it suddenly says credit score. That seems oddly specific for such a generally named function.

Reply:
Fair point. I've added some documentation